docs/content/CounterLabel.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: CounterLabel
+tags: ["pill"]
```

Comment:
Sorry, these don't need to be in an array, you can list it in this format:
```suggestion
tags: pill
```

Reply:
Awesome, thanks! I was just trying to follow the [Gatsby docs](https://github.com/primer/components/pull/586#discussion_r334171629), but if this works too, even better. :)